### PR TITLE
Added Jumpstart API example for Object Detection

### PR DESCRIPTION
**Pull request by vivekmadan2**
Added a notebook to demonstrate how to do object detection using Jumpstart

Related PR: https://github.com/aws/amazon-sagemaker-examples/pull/3205
Reason for new PR: Changing the folder where the notebook is located.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [ x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [ x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ x] I have tested my notebook(s) and ensured it runs end-to-end
- [ x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jkroll-aws — APPROVED** [review]

File: introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb [hunk 1]
```diff
@@ -0,0 +1,797 @@
+{
```
Comment (evakravi) [inline introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb:1]: <p>Don't we need to delete the endpoint config and model too? Or does the <code>predictor</code> do this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (vivekmadan2) [reply introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb:1]: endpoint config and model don't actually use any resources (other than storage which is relatively cheap) and are typically not deleted. But, adding model deletion in this and all the subsequent notebooks to be extra cost efficient.

File: introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
```
Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb:1]: <p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_object_detection/Amazon_JumpStart_Object_Detection.ipynb:1]: <p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Review by atqy — APPROVED** [review]

**Pull request merged**